### PR TITLE
using write_unitted

### PR DESCRIPTION
**Pull request by jread-usgs**
this is staging, but related to content in #68

### CODE REVIEW

File: R/stage_nldas_ts.R [hunk 1]
```diff
@@ -28,20 +30,23 @@ stage_nldas_ts <- function(sites, variable_name, times, ...){
   for (i in 1:length(sites)){
     location <- filter(site_data, site_no == nwis_sites[i]) %>%
       select(dec_lat_va, dec_long_va) %>% 
-      summarize(lon = mean(dec_long_va, na.rm = T), dlat = mean(dec_lat_va, na.rm = T))
+      summarize(lon = mean(dec_long_va, na.rm = T), lat = mean(dec_lat_va, na.rm = T))
```
Comment (jread-usgs) [inline R/stage_nldas_ts.R]: some sites return two lat and two long

File: R/stage_nldas_ts.R [hunk 2]
```diff
@@ -28,20 +30,23 @@ stage_nldas_ts <- function(sites, variable_name, times, ...){
...
     lon_lat[1:2, i]  = as.numeric(location)
 
   }
-  lon_lat_df <- as.data.frame(lon_lat)
-  names(lon_lat_df) <- sites
+  rmv_sites <- is.na(lon_lat[1,]) | is.na(lon_lat[2,])
```
Comment (jread-usgs) [inline R/stage_nldas_ts.R]: currently a fix for https://github.com/USGS-CIDA/stream_metab_usa/issues/8

File: R/stage_nwis_ts.R [hunk 3]
```diff
@@ -33,12 +34,12 @@ stage_nwis_ts <- function(sites, variable_name, times, ...){
       site <- un_sites[i]
       site_data <- filter(nwis_data, site_no == site) %>%
         mutate(DateTime = as.POSIXct(dateTime, tz = tz_cd)) %>%
-        select(DateTime, matches(tail(names(nwis_data),1)), -ends_with("_cd"))
+        select(DateTime, matches(tail(names(nwis_data),1)), -ends_with("_cd")) %>%
+        setNames(c("DateTime",ts_name)) %>%
+        u(c('UTC','units'))
```
Comment (jread-usgs) [inline R/stage_nwis_ts.R]: needs #80

File: R/stage_nldas_ts.R [hunk 4]
```diff
@@ -50,14 +55,14 @@ stage_nldas_ts <- function(sites, variable_name, times, ...){
...
-    names(site_data) <- c('DateTime',p_code)
-    site_data <- u(site_data, c(NA, units))
-    file_handle <- sprintf('%s_%s.tsv',site,ts_name)
-    write_unitted(site_data, file = file_handle, ...)
+    
+    site_data <- select(site_data, -units) %>%
+      setNames(c('DateTime',p_code)) %>%
+      u(c("UTC", units))
+    
+    file_handle <- sprintf('%s/nwis_%s_%s.tsv', folder, site, ts_name)
```
Comment (aappling-usgs) [inline R/stage_nldas_ts.R]: should this be nldas instead of nwis?

File: R/stage_nwis_ts.R [hunk 5]
```diff
@@ -2,30 +2,31 @@
...
+#'files <- stage_nwis_ts(sites = c("nwis_06893820","nwis_01484680"), variable = "doobs",
 #'                  times = c('2014-01-01','2014-02-01'))
-#'files <- stage_nwis_ts(sites = get_sites(), variable_name = "par",
+#'\dontrun{
+#'files <- stage_nwis_ts(sites = get_sites(), variable = "par",
 #'                  times = c('2014-01-01', '2014-02-01'))
 #'}
 #'@export
-stage_nwis_ts <- function(sites, variable_name, times, ...){
+stage_nwis_ts <- function(sites, variable, times, folder = tempdir(), verbose = FALSE, ...){
```
Comment (aappling-usgs) [inline R/stage_nwis_ts.R]: i like the folder option

Comment (aappling-usgs) [inline R/stage_nwis_ts.R]: for the record, looks like verbose doesn't do anything yet

File: R/stage_nldas_ts.R [hunk 6]
```diff
@@ -2,24 +2,26 @@
 #'@description get data from nldas and return a file handle
 #'
 #'@param sites a character vector of valid NWIS site IDs
-#'@param variable_name short name of variable \code{\link{get_ts_variables}}
+#'@param variable short name of variable \code{\link{get_ts_variables}}
 #'@param times a length 2 vector of POSIXct dates
-#'@param ... additional arguments passed to \code{\link[geoknife]{geoknife}}
+#'@param folder a folder to place the file outputs in (defaults to temp directory)
+#'@param verbose provide verbose output (currently not implemented)
```
Comment (jread-usgs) [inline R/stage_nldas_ts.R]: yep, not implemented for either of the two.

**Comment by aappling-usgs** [comment]
yeah, this all looks good

**Comment by jread-usgs** [comment]
thx

**Pull request merged**